src/components/datagrid/controls/display_selector.tsx

```diff
@@ -318,6 +360,17 @@ export const useDataGridDisplaySelector = (
             )}
           </EuiI18n>
         )}
+        {showResetButton && (
+          <EuiPopoverFooter>
+            <EuiButtonEmpty
+              size="xs"
+              onClick={resetToInitialState}
+              data-test-subj="resetDisplaySelector"
+            >
+              {resetButtonLabel}
+            </EuiButtonEmpty>
+          </EuiPopoverFooter>
+        )}
```

Comment:
I know it's more markup but I'd love to align this to the right of the popover similar to the sort popover.
```suggestion
        {showResetButton && (
          <EuiPopoverFooter>
            <EuiFlexGroup justifyContent="flexEnd" responsive={false}>
              <EuiFlexItem>
                <EuiButtonEmpty
                  flush="both"
                  size="xs"
                  onClick={resetToInitialState}
                  data-test-subj="resetDisplaySelector"
                >
                  {resetButtonLabel}
                </EuiButtonEmpty>
              </EuiFlexItem>
            <EuiFlexGroup>
          </EuiPopoverFooter>
        )}
```

Reply:
For sure, makes sense!